Testscripts/Linux/dpdkUtils.sh

```diff
@@ -167,8 +167,9 @@ function Install_Dpdk () {
 			ssh "${1}" ". utils.sh && install_epel"
 			ssh "${1}" "yum -y groupinstall 'Infiniband Support' && dracut --add-drivers 'mlx4_en mlx4_ib mlx5_ib' -f && systemctl enable rdma"
 			check_exit_status "Install Infiniband Support on ${1}" "exit"
-			ssh "${1}" "(grep 7.5 /etc/redhat-release || grep 7.6 /etc/redhat-release) && curl https://partnerpipelineshare.blob.core.windows.net/kernel-devel-rpms/CentOS-Vault.repo > /etc/yum.repos.d/CentOS-Vault.repo"
+			ssh "${1}" "(grep -E '7.5|7.6|7.8' /etc/redhat-release) && curl https://partnerpipelineshare.blob.core.windows.net/kernel-devel-rpms/CentOS-Vault.repo > /etc/yum.repos.d/CentOS-Vault.repo"
```

Comment:
what about 7.x that's newer than 7.8 in the future? should we change step by step? any inconvenience if we place 7.[\d+] here?

Reply:
7.7, 8.0, 8.1 and 8.2 it has libmnl-devel package in default repository.

Reply:
Thanks, @LiliDeng This kinds of dependency repo change has been headache, I wished those were consistent, but I believe each distro repo management has new plan. So we should check the same problem in every new version release. Thanks for taking care of it.